app/core/config.py

```diff
@@ -63,3 +63,4 @@
 NF_WORKSPACE_ID = config("NF_WORKSPACE_ID")
 
 UPLOAD_DIRECTORY = config("UPLOAD_DIRECTORY", default="/tmpdir")
+ENSEMBL_WEB_METADATA_API = config("ENSEMBL_WEB_METADATA_API")
```

Comment:
Checkout my PR for this env variable WEB_METADATA_API.

Reply:
Would you like me to use the same name?

Reply:
Yes please, It makes sense to use the same config name.